readme.md

```diff
@@ -10,6 +10,15 @@
 docker build -t deepnote/python:<TAG>  --build-arg FROM_PYTHON_VERSION=<some_version> ./python
 ```
 
+## Conda images
+For every Python version (see above), we also build a `-conda` variant, which includes [Miniconda](https://docs.conda.io/en/latest/miniconda.html) installation.
+The Dockerfile for this is at [./python/Dockerfile-conda](https://github.com/deepnote/environments/blob/main/python/Dockerfile-conda).
+
+### How to build
+```
+docker build --build-arg FROM_PYTHON_VERSION=<some_version> -t deepnote/python:<some_version>-conda ./python --file .python/Dockerfile.conda
```

Comment:
```suggestion
docker build --build-arg FROM_PYTHON_VERSION=<some_version> -t deepnote/python:<some_version>-conda ./python --file ./python/Dockerfile.conda
```

Reply:
Sorry. I missed this one. Added it in https://github.com/deepnote/environments/commit/038d372bb06429ec72f2fc0fdc772b759342a51a